gno.land/cmd/genesis/utils.go

```diff
@@ -0,0 +1,80 @@
+package main
```

Comment:
I'm not a big fan of utils files. IMHO they are "everything can go there" boxes.

Reply:
Dropped:

[b7f8b61](https://github.com/gnolang/gno/pull/1276/commits/b7f8b610ea53da91687cc47471c434b3a7bfdd5d)